terraform/helm/validator/files/configs/fullnode.yaml

```diff
@@ -21,6 +21,8 @@ execution:
 
 storage:
   backup_service_address: "0.0.0.0:6186"
+  storage_pruner_config:
+    default_prune_window: {{ .Values.validator.config.default_prune_window | default 10000000 }}
```

Comment:
Let's make this a billion just to be on the safe side?

Reply:
Just to be clear, 10 M is the current default value in code, we want to bump it to 1 B over here.

Reply:
@sitalkedia the override will happen on the terraform side, each testnet can customize their own value from there, here I just matched the code default

Reply:
Ah, I see. Thanks for clarifying.

Reply:
@sitalkedia the override PR  https://github.com/aptos-labs/internal-ops/pull/147